aws_cloudtrail_rules/aws_codebuild_made_public.py

```diff
@@ -0,0 +1,14 @@
+from panther import lookup_aws_account_name
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return event['eventName'] == 'UpdateProjectVisibility' and\
+        deep_get(event, 'requestParameters', 'projectVisibility') == 'PUBLIC_READ'
```

Comment:
Did you run `make fmt`? Mostly curious because I think `black` will change this (prefers parentheses to backslashes)

```suggestion
    return (event['eventName'] == 'UpdateProjectVisibility' and
        deep_get(event, 'requestParameters', 'projectVisibility') == 'PUBLIC_READ')
```

Reply:
I did not - this was my attempt to make pylint/circleci happy. Will do that next time. Committing your change, thanks!